### PR TITLE
fix broken links due to sub-dir structuring... 

### PR DESCRIPTION
**Pull request by Brilator**

### CODE REVIEW

File: src/docs/README.md [hunk 1]
```diff
@@ -13,30 +13,26 @@ todo:
...
-        - [Visible or Hidden](#visible-or-hidden)
-- [Content Design Principles](content-design-principles)
-    - [References](#references)
-    - [Structure and Format](#structure-and-format)
-    - [Images](#images)
-    - [File Name Requirements](#file-name-requirements)
-    - [Addressing Readers](#addressing-readers)
-    - [Language](#language)
-
+- [Common Errors](#common-errors)
```
Comment (Freymaurer) [inline src/docs/README.md:23]: Why delete the h1 headers from TOC?

File: src/docs/tutorials/DataPlantOnboarding.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+layout: docs
```
Comment (Freymaurer) [inline src/docs/tutorials/DataPlantOnboarding.md:2]: I like the `author_orcid` and `author_github` 👍

Comment (Brilator) [reply src/docs/tutorials/DataPlantOnboarding.md:2]: They made it to the yaml in README last week already :) 

https://github.com/nfdi4plants/nfdi4plants.knowledgebase/blob/main/src/docs/README.md

File: src/docs/README.md (hunk 1 above)
Comment (Brilator) [reply src/docs/README.md:23]: That's my default of the vscode extension updating the TOC... 
I've updated the h1's to h2's, etc. to keep title as the only h1. 
The README will need some overall revision and re-structuring.

Comment (Freymaurer) [reply src/docs/README.md:23]: > I've updated the h1's to h2's, etc. to keep title as the only h1.

I don't really see why we should do that. I think using h1-h3 is perfectly fine and creates nicely organized tocs.

But if you prefer it that way we can do that.

Comment (Brilator) [reply src/docs/README.md:23]: Of course we can use h1, but then the first heading used should be an h1. 
The way it was, we had some h2's first and then at some point in the middle two h1's

**Pull request merged**